api/v1/endpoints/queries.sql

```diff
@@ -65,7 +65,9 @@ WITH object_data AS (
         JOIN course_instances AS ci ON (ci.id = a.course_instance_id)
         JOIN assessment_sets AS aset ON (aset.id = a.assessment_set_id)
         JOIN assessment_instances AS ai ON (ai.assessment_id = a.id)
-        JOIN users AS u ON (u.user_id = ai.user_id)
+        LEFT JOIN group_info($assessment_id) AS gi ON (gi.id = ai.group_id)
```

Comment:
We can't assume that `$assessment_id` is not-NULL here, as you can see from the WHERE statement below: https://github.com/PrairieLearn/PrairieLearn/blob/5b997eabb69a0b5707f6b0e2ab2d615069455756/api/v1/endpoints/queries.sql#L71

Instead you should use `a.id` which is guaranteed to be not-NULL.